app/src/main/java/org/hisp/dhis/android/dataentry/form/section/EditTextHintCache.java

```diff
@@ -0,0 +1,7 @@
+package org.hisp.dhis.android.dataentry.form.section;
+
+public interface EditTextHintCache {
```

Comment:
Implementing the whole class just for caching prompts seems quite redundant. Why not just retrieve prompt within ViewHolder's constructors and cache it in the field?